### PR TITLE
feat：add fast-rcnn/mask-rcnn/yolox  partition_onnxruntime_static export

### PR DESCRIPTION
**Pull request by Uhao-P**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

add fast-rcnn/mask-rcnn/yolox  partition_onnxruntime_static export

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdeploy/codebase/mmdet/models/dense_heads/yolox_head.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def yolox_head__get_bboxes(ctx,
             represents the class label of the corresponding box.
     """
     # mark pred_maps
-    @mark('yolo_head', inputs=['cls_scores', 'bbox_preds', 'objectnesses'])
+    @mark('yolox_head', inputs=['cls_scores', 'bbox_preds', 'objectnesses'])
```
[the remark this answers is not in the document]
Comment (Uhao-P) [reply mmdeploy/codebase/mmdet/models/dense_heads/yolox_head.py:52]: I didn't try yolo_head. I don't have @mark in yolox_head.py yet when I try to export yolox. I think it makes more sense for different heads to have unique names.

[the remark this answers is not in the document]
Comment (AllentDan) [reply mmdeploy/codebase/mmdet/models/dense_heads/yolox_head.py:52]: but it cannot be reused other yolo models then.

[the remark this answers is not in the document]
Comment (Uhao-P) [reply mmdeploy/codebase/mmdet/models/dense_heads/yolox_head.py:52]: I think you are right. I'll test it

**Comment by AllentDan** [comment]
Hi, @Uhao-P. After a discussion in our group, these partition configs are not so suitable in our codebase since they are customized and not following the style of other configs in MMDeploy.

We welcome you to keep these features open-source and add them to the [OpenMMLab ecosystem](https://openmmlab.com/ecosystem).

**Comment by OpenMMLab-Assistant005** [comment]
Hi @Uhao-P ！We are grateful for your efforts in helping improve this open-source project during your personal time. 
Welcome to join OpenMMLab Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request closed without merging**